.semaphore/semaphore.yml

```diff
@@ -24,7 +24,7 @@ global_job_config:
   prologue:
     commands:
       - checkout
-      - sem-version java 11
+      - sem-version java 17
```

Comment:
Temp fix as https://github.com/confluentinc/kafka-streams-examples/commit/ae49f5fe85f124e138e722c8e92840fdeddbde97 did not make it into `master` yet.

\cc @hk10111 -- hope we can resolve the merge conflicts soon... Or merging this PR will create more conflicts.